src/sentry/models/tempest_credentials.py

```diff
@@ -0,0 +1,35 @@
+from django.conf import settings
+from django.db import models
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import DefaultFieldsModel, FlexibleForeignKey, region_silo_model
+from sentry.db.models.fields.hybrid_cloud_foreign_key import HybridCloudForeignKey
+
+
+class MessageType(models.TextChoices):
+    ERROR = "error"
+    WARNING = "warning"
+    SUCCESS = "success"
+    INFO = "info"
+
+
+@region_silo_model
+class TempestCredentials(DefaultFieldsModel):
```

Comment:
I'm not too familiar with what tempest is, but if it is an entirely new product component/feature/service you may want to use a django-app. Using a django-app would give you a separate migration history which reduces contention on the head of migration history, and also sets you up with a directory structure to put endpoints, tasks and serializers.

Reply:
At first i was hesitant to create a new app for it, but with everything you have mentioned, it def makes more sense to have it in a separate app